index.html

```diff
@@ -1107,8 +1107,35 @@
 
 ## Principles for Notifications and Interruptions
 
-<aside class="issue">Connect intrusiveness to [=autonomy=], and write some principles.</aside>
+<aside class="issue">This is related to [=autonomy=].</aside>
```

Comment:
You've mentioned autonomy in the next paragraph, so maybe we can remove this issue.